core/types/dynamic_fee_tx.go

```diff
@@ -444,6 +443,8 @@ func (tx DynamicFeeTransaction) AsMessage(s Signer) (Message, error) {
 		accessList: tx.AccessList,
 		checkNonce: true,
 	}
+	msg.gasPrice.SetFromBig(header.BaseFee)
+	msg.gasPrice.Add(&msg.gasPrice, tx.Tip)
```

Comment:
In case that tx.Tip + block.baseFee > tx.FeeCap then the gasprice should be tx.FeeCap.

Reply:
Ok, thank you :)